docs/checks.md

```diff
@@ -261,7 +261,7 @@ participants.
 
```

Comment:
This is an is auto-generated file and shouldn't be modified directly.  You will need to change `docs/checks/internal/checks.yaml` and run `make generate-docs` instead.